tools/build-tools/package.json

```diff
@@ -63,6 +64,7 @@
     "@types/semver": "^7.1.0",
```

Comment:
type types/mocha entry got deleted from the original change which is where the build error is coming from